docs/sources/reference/commandline/cli.md

```diff
@@ -56,18 +58,18 @@ The Docker command line stores its configuration files in a directory called
 `.docker/config.json` file to control certain aspects of how the `docker`
 command behaves.
 
-Currently, you can modify the `docker` command behavior using environment 
-variables or command-line options. You can also use options within 
-`config.json` to modify some of the same behavior.  When using these 
-mechanisms, you must keep in mind the order of precedence among them. Command 
-line options override environment variables and environment variables override 
+Currently, you can modify the `docker` command behavior using environment
+variables or command-line options. You can also use options within
+`config.json` to modify some of the same behavior. When using these
+mechanisms, you must keep in mind the order of precedence among them. Command
+line options override environment variables and environment variables override
 properties you specify in a `config.json` file.
 
 The `config.json` file stores a JSON encoding of a single `HttpHeaders`
-property. The property specifies a set of headers to include in all
-messages sent from the Docker client to the daemon. Docker does not try to
-interpret or understand these header; it simply puts them into the messages.
-Docker does not allow these headers to change any headers it sets for itself.
+property. The property specifies a set of headers to include in all messages
+sent from the Docker client to the daemon. Docker does not try to interpret or
+understand these header; it simply puts them into the messages. Docker does
+not allow these headers to change any headers it sets for itself.
```

Comment:
The antecedent it a bit unclear.  Maybe just:

"The headers that Docker sets for itself cannot be changed."